invenio_records_rest/views.py

```diff
@@ -74,6 +76,11 @@ def validation_error(error):
         """Catch validation errors."""
         return RESTValidationError().get_response()
 
+    @blueprint.errorhandler(RequestError)
+    def elasticsearch_badrequest_error(error):
+        """Catche problems in ElasticSearch queries."""
+        return BadRequest(description=error.error).get_response()
```

Comment:
I checked the exceptions in ``invenio_rest.errors`` but there is no one that fits properly here. 

About the description, what you are getting in this case is a ``search_phase_execution_exception``. Maybe not in all cases the error is going to be the same. I can rewrite the error description, ~~but isn't better to use ``{% trans %}`` in the view part with the error response?~~Ok, I understand that Angular is making things more complicated here.

I am checking now why Zenodo is not showing the error descriptions. It is just showing ``Error:`` and nothing else
![zenodo_error](https://cloud.githubusercontent.com/assets/6481083/15737223/45ce6b7a-28a5-11e6-8782-a8617232be42.png)
: